= *MemPanG26*
:figure-caption!:
:toc-title: MemPanG26
:toc: left
:toclevels: 2

Computational pangenomics course, conference, and biohackathon in Memphis, TN, exploring the cutting edge of pangenomes, biology, methods, software, and artificial intelligence (AI).

image::images/DRB1-3123.fa.gz.pggb-E-s5000-l15000-p80-n10-a0-K16-k8-w50000-j5000-e5000-I0-R0-N.smooth.chop.og.lay.draw_mqc.CROP.png[400]
image::images/bridge.png[400]


== *TO CONFIRM: Important Dates*

- Practical course/Conference/Biohackathon Registration Deadline: *1st April 2026*
- Selected Participant will be notified by: *7th April 2026*
- Payment Deadline: *21th April 2026*
- Practical course / Workshop: *11th-12th May 2026*
- Conference: *13th May 2026*
- Biohackathon: *14th-15th May 2026*
////
== *OPEN CALL FOR SPEAKERS - MemPanG26*

We are thrilled to announce an open invitation to experts and enthusiasts in the field of pangenome research to be a part of our upcoming MemPanG26.
This significant event is scheduled for May 13th, 2026, and aims to delve into the multifaceted world of pangenome research.
The submission deadline for talk proposals is *April 15th, 2026*.

*Focus Areas:* Our conference will cover a broad range of topics, including but not limited to, methods in pangenome research, genome assembly, data visualization, artificial intelligence (AI), and collaborative projects.

*Diverse Perspectives:* We are looking for speakers who can provide insights from different species contexts and who can discuss topics that lie at the intersection of pangenome methods and biology. We are particularly interested in hearing about your unique work and perspectives in pangenome research.

*Duration:* Each talk should be approximately 20 minutes, followed by a Q&A session, amounting to a total of 25 minutes.

*Submission:* We kindly request you to https://docs.google.com/forms/d/e/1FAIpQLSe2mnJ_kZm_I2j5lIBoLGL-hLW4mxzHpphvWomLbsul4C8goA/viewform?usp=sf_link[forward the title and abstract of your talk] upon confirmation of your participation.
You can see abstract examples at https://pangenome.github.io/MemPanG26/abstract_examples.html.

*Location:* The event will be held in Memphis.

*Conference pre-registration for speakers:* https://docs.google.com/forms/d/e/1FAIpQLSe2mnJ_kZm_I2j5lIBoLGL-hLW4mxzHpphvWomLbsul4C8goA/viewform?usp=sf_link

Let's make this a fun day!

MemPanG26 organizers
////
== *TO CONFIRM: Venues*

=== Practical course / Workshop
*Freeman Auditorium at the Hamilton Eye Institute* +
930 Madison Ave, +
3rd floor, +
Memphis, TN 38103 +
United States +
-> https://www.google.com/maps/place/930+Madison+Ave,+Memphis,+TN+38103/@35.1405526,-90.0321798,17z/data=!3m1!4b1!4m6!3m5!1s0x87d57e65083437ab:0x975bb3c7cbb47cf0!8m2!3d35.1405526!4d-90.0296049!16s%2Fg%2F11c26jf2ws?entry=ttu[See the venue on the map]

=== Conference
*Freeman Auditorium at the Hamilton Eye Institute* +
930 Madison Ave, +
3rd floor, +
Memphis, TN 38103 +
United States +
-> https://www.google.com/maps/place/930+Madison+Ave,+Memphis,+TN+38103/@35.1405526,-90.0321798,17z/data=!3m1!4b1!4m6!3m5!1s0x87d57e65083437ab:0x975bb3c7cbb47cf0!8m2!3d35.1405526!4d-90.0296049!16s%2Fg%2F11c26jf2ws?entry=ttu[See the venue on the map]

=== Biohackathon
*Room 101, UTHSC Translational Research Science Building* +
71 S Manassas St, +
Memphis, TN 38103 +
United States +
-> https://maps.app.goo.gl/QpQgUyRyjzjxP1vM9[See the venue on the map]

=== Hotels in Memphis
- link:https://goo.gl/maps/arRZWkjPCNBAFfCf9[La Quinta Hotel Memphis Downtown]
- link:https://goo.gl/maps/Dwf9LgHeJLXsAJcf7[Double Tree Hotel]
- link:https://goo.gl/maps/h5A6LpmToTD7DepH8[River Inn at Harbor Town Hotel]
- link:https://goo.gl/maps/o7XrW3DZHTNqXXT26[Comfort Inn]
- link:https://goo.gl/maps/e365A6rNjZUPvFqRA[Hampton Inn]
- link:https://goo.gl/maps/rjdSg46kZFPsmKxS7[Hyatt Centric Beale Street]
- link:https://goo.gl/maps/e4BcvEabefsqEhC4A[HU Hotel]
- link:https://goo.gl/maps/Erq5cwVtM4hh8c8e7[Sheraton Memphis Downtown Hotel]

=== Transportation in Memphis

If you stay in Downtown, the Medical District, South City, or New Chicago, you can reach the event venue by getting on-demand, convenient, and affordable rides with link:https://city.ridewithvia.com/groove-on-demand[Groove On-Demand].

== *About the practical course / workshop*

*Sunday, May 11, 2026 - Monday, May 12, 2026* +
The use of a single reference genome in bioinformatics can lead to reference bias and miss important information about genome variability and relationships. New assembly methods have made it easier to generate high-quality complete genome assemblies, and using a pangenome graph that expresses many genomes and their mutual alignment can help address these issues.
This practical course / workshop will focus on building such graphs from whole genome assemblies and using them in various downstream applications in comparative genomics, evolution, variation analysis, sequence alignment, and phenotype association.

Participants will learn about pangenome concepts and gain practical experience building and analyzing pangenome graphs. They will apply these methods to complex research questions that require understanding the relationships between multiple genomes or accounting for variability when analyzing new genomes.
By the end of the practical course / workshop, participants will have a strong understanding of pangenome methods based on whole genome assemblies.

== *About the conference*

*Tuesday, May 13, 2026* +
We will gather for an in-person conference on the day after the practical course / workshop.
This will bring together practical course / workshop participants and virtual and in-person speakers who are actively working on diverse aspects of pangenome research.
The virtual meeting will be open to the public.
Speakers will cover topics at the intersection of pangenome methods and biology, including a focus on basic methods, data structures, genome assembly, data visualization, and public collaborative pangenome projects.
We expect to draw speakers from a diverse set of species contexts.

== *About the biohackathon*

*Wednesday, May 14, 2026 - Thursday, May 15, 2026* +

We have a focus on software.
We will work with/on pangenomic tools (such as link:https://github.com/pangenome/pggb/[PGGB], link:https://github.com/waveygang/wfmash/[WFMASH],  link:https://github.com/pangenome/impg/[IMPG],link:https://github.com/ekg/seqwish/[SEQWISH], link:https://github.com/pangenome/odgi/[ODGI], link:https://github.com/pangenome/smoothxg/[SMOOTHXG], link:https://github.com/vgteam/vg[VG], link:https://github.com/vcflib/vcflib/[VCFLIB], link:https://github.com/genetics-statistics/GEMMA/[GEMMA]) with software project leadership.
*You are welcome to bring your own projects!*

== *People*

=== Practical Course / Workshop Instructors


==== Erik Garrison - Associate Professor

.[purple]#University of Tennessee Health Science Center, Memphis, TN, US#

image:images/erik.jpeg[erik,200,role="right"] Genomicist with a quantitative social science background. I build methods that let us understand the precise relationships between thousands of genomes. In these, the genome is encoded in a graph that may represent a population sample of individuals from the same species, a metagenome, the diploid genome of a single individual, or any other useful collection of sequences.

image:images/Octicons-mark-github.svg[git,30] https://github.com/ekg[GitHub]

==== Andrea Guarracino - Assistant Professor

.[purple]#University of Tennessee Health Science Center, Memphis, TN, US#

image:images/andrea.jpeg[erik,200,role="right"] Computer scientist dedicated to developing and applying new methods to analyze pangenomes.

image:images/Octicons-mark-github.svg[git,30] https://andreaguarracino.github.io/[Website]

////

'''

==== Simon Heumos - PhD student

.[purple]#Quantitative Biology Center (QBiC), University of Tübingen, Tübingen, DE | Biomedical Data Science, Department of Computer Science, University of Tübingen, Tübingen, DE#

image:images/simon.png[simon,200,role="right"]

Bioinformatician by training. I am exploring methods to calculate and visualize pangenome graph layouts. These are key steps in pangenome graph construction and analysis pipelines. Cluster-scalable pangenomics is the next step.

image:images/Octicons-mark-github.svg[git,30] https://github.com/subwaystation[GitHub]

'''

////

'''

=== Helpers

////

==== Jan-Niklas Schmelzle - PhD student

.[purple]#ECE PhD Student, Cornell University, US#
image:images/niklas.jpeg[niklas,200,role="right"]

+
+
+

image:images/Octicons-mark-github.svg[git,30] https://scholar.google.com/citations?user=m_vTR7YAAAAJ&hl=en[Google Scholar]

==== Arun Isaac - Postdoctoral Scholar

.[purple]#University College London, London, UK#
image:images/arun.jpg[arun,200,role="right"]

+
+
+

image:images/Octicons-mark-github.svg[git,30] https://aruni.systemreboot.net[Website]

////

////

==== Flavia Villani - PhD student

.[purple]#University of Tennessee Health Science Center, Memphis, TN, US#
image:images/Flavia.JPG[flavia,200,role="right"] I have a master degree in Medical Biotechnology from the University of Naples Federico II. Currently, I am a PhD student at the University of Tennessee Health Science Center. I am building the pangenome of model organisms (inbred mice and rats) using a combination of second and third generation sequence data.

image:images/Octicons-mark-github.svg[git,30] https://github.com/Flavia95[GitHub]

'''

////

////

- link:https://scholar.google.com/citations?user=C0AkpcEAAAAJ&hl=en[Mohab Helmy]
- link:https://scholar.google.ch/citations?hl=en&user=yM9CokEAAAAJ[Ragnar Groot Koerkamp]
- link:https://l.iyi.fan/[Yifan Li]
- link:https://scholar.google.com/citations?user=Bq50aCUAAAAJ&hl=en[Adrian Sampson]
- link:https://scholar.google.com/citations?user=UZLC4TYAAAAJ&hl=en[Giulia Guidi]
- link:https://scholar.google.com/citations?user=HyOMCyUAAAAJ&hl=en[Brandi Cantarel]
- link:https://aruni.systemreboot.net[Arun Isaac]

////

'''

=== Organizers

- link:https://andreaguarracino.github.io/[Andrea Guarracino], University of Tennessee Health Science Center, Memphis, TN, US
- link:http://hypervolu.me/~erik/erik_garrison.html[Erik Garrison], University of Tennessee Health Science Center, Memphis, TN, US
- link:https://github.com/Flavia95[Flavia Villani], University of Tennessee Health Science Center, Memphis, TN, US
- link:https://thebird.nl/[Pjotr Prins], University of Tennessee Health Science Center, Memphis, TN, US
- link:https://www.uthsc.edu/faculty/profile/?netid=vcolonna[Vincenza Colonna], University of Tennessee Health Science Center, Memphis, TN, US
- link:https://uthsc.edu/search/detail.php?id=T100344325[Tamara Brock], University of Tennessee Health Science Center, Memphis, TN, US
// - link:https://aruni.systemreboot.net[Arun Isaac], University College London, London, UK
// - link:https://www.christinaboucher.com/[Christina Boucher], University of Florida, Gainesville, FL, US

image:images/UTHSC.png[uthsc,300]

University of Tennessee Health Science Center, Department of Genetics, Genomics and Informatics

== *Program*

=== Practical course / Workshop

*Time:* Monday-Tuesday, 11th-12th May 2026 +

*Place*: link:https://www.google.com/maps/place/930+Madison+Ave,+Memphis,+TN+38103/@35.1405526,-90.0321798,17z/data=!3m1!4b1!4m6!3m5!1s0x87d57e65083437ab:0x975bb3c7cbb47cf0!8m2!3d35.1405526!4d-90.0296049!16s%2Fg%2F11c26jf2ws?entry=ttu[930 Madison Ave, Freeman Auditorium at the Hamilton Eye Institute, 3rd floor]

// [options="header", cols="2,1,2,4"]
// |===
// |Day | Time | Speaker(s) | Topic

// |Sunday, May 11, 2026 | 09:30-10:00  | Erik Garrison | Presentation: Introduction to pangenomics
// || 10:00-10:30 | Everyone | Practical: Build your first pangenome graphs
// || *10:30-11:00* | *Everyone* | *Coffee break*
// || 11:00-12:30 | Everyone | Practical: PGGB
// || *12:30-14:00* | *Everyone* | *Lunch (provided)*
// || 14:00-14:30 | Andrea Guarracino | Presentation: Understanding pangenomes
// || 14:30-15:30 | Everyone | Practical: ODGI commands
// || *15:30-16:00* | *Everyone* | *Coffee break*
// || 16:00-16:45 | Everyone | Practical:  ODGI commands
// || 16:55-17:00 | Everyone | Q&A, Day 1 Survey
// |===

// [options="header", cols="2,1,2,4"]
// |===
// |Day | Time | Speaker(s) | Topic
// |Monday, May 12, 2026 | 09:30-09:45 | Everyone | Discussion about previous day activities
// || 09:45-10:00 | Erik Garrison | Presentation: Sequence partitioning
// || 10:00-10:30 | Everyone | Practical: Reference-guided partitioning
// || *10:30-11:00* | *Everyone* | *Coffee break*
// || 11:00-12:30 | Everyone | Practical: Reference-free partitioning
// || *12:30-14:00* | *Everyone* | *Lunch (provided)*
// || 14:00-14:30 | Andrea Guarracino | Presentation: Implicit pangenomics and pangenome genotyping
// || 14:30-16:00 | Everyone | Practical: Implicit pangenome graphs
// || *16:00-16:30* | *Everyone* | *Coffee break*
// || 16:00-16:45 | Everyone | Practical: Pangenome-based genotyping
// || 16:55-17:00 | Everyone | Q&A, Day 2 Survey
// || 17:00-17:05 | TBD | Closure
// |===

=== Conference

*Time:* Wednesday, 13 May 2026 +

*Place*: link:https://www.google.com/maps/place/930+Madison+Ave,+Memphis,+TN+38103/@35.1405526,-90.0321798,17z/data=!3m1!4b1!4m6!3m5!1s0x87d57e65083437ab:0x975bb3c7cbb47cf0!8m2!3d35.1405526!4d-90.0296049!16s%2Fg%2F11c26jf2ws?entry=ttu[930 Madison Ave, Freeman Auditorium at the Hamilton Eye Institutee, 3rd floor]

// [options="header", cols="2,2,5,2"]
// |===
// |Chair | Time | Talk | Speaker

// || 09:25 - 09:30 | Conference Opening | link:https://www.uthsc.edu/faculty/profile/?netid=rwilli10/[Robert Williams]

// |link:https://andreaguarracino.github.io/[Andrea Guarracino] | 09:30 - 10:00 | Visualize Whole Genome Multiple Sequence Alignments in NCBI’s New Multiple Comparative Genome Viewer (MCGV) | Sanjida Rangwala
// || 10:00 - 10:30 | PangyPlot: An Interactive Graph Viewer Designed for Reference Pangenomes | Scott Mastromatteo
// || 10:30 - 11:00 | Novel Chromosomal Rearrangements and Their Role in Speciation in Primates | Brooklynn Scott
// || *11:00 - 11:30* | *Coffee break* | *Coffee*

// | link:https://scholar.google.com/citations?user=m_vTR7YAAAAJ&hl=en/[Jan-Niklas Schmelzle] | 11:30 - 12:00 | Pangenome graph resolves the dynamic evolution of the plant immune system | Gautam Shirsekar
// || 12:00 - 12:30 | Diverse graph-based approaches for complex collections of plant genomes | Avril Harder
// || 12:30 - 13:00 | Integrating genetic ancestry, environmental exposure and health equity | Vincenza Colonna
// || *13:00 - 14:00* | *Lunch (provided)* | *Food*

// |link:https://www.linkedin.com/in/shelbysolomondarnell/?originalSubdomain=ke[Shelby Solomon Darnell] | 14:00 - 14:30 | Efficient Grammar Compression via RLZ-based RePair | Rahul Varki
// || 14:30 - 15:00 | Disentangling reticulate evolution in Mediterranean Wall Lizards with a super pangenome | Natalia Zajac
// || 15:00 - 15:05 | Conference Closing | Vincenza Colonna
// |===

=== Biohackathon

*Time:* Thursday-Friday, 14th-15th May 2026 +

*Place:* link:https://maps.app.goo.gl/QpQgUyRyjzjxP1vM9[Room 101, UTHSC Translational Research Science Building]

// [options="header", cols="2,1,2,4"]
// |===
// |Day | Time | Speaker(s) | Topic

// |Wednesday, May 14, 2026 | 09:30-10:30 | Everyone | Hacking
// || 10:30-11:00 | Everyone | Coffee break
// || 11:00-12:30 | Everyone | Hacking
// || *12:30-14:30* | *Everyone* | *Lunch*
// || 14:30-15:30 | Everyone | Hacking
// || 15:30-16:00 | Everyone | Coffee break
// || 16:00-17:00 | Everyone | Hacking
// |===

// [options="header", cols="2,1,2,4"]
// |===
// |Day | Time | Speaker(s) | Topic
// |Thursday, May 15, 2026 | 09:30-10:30 | Everyone | Hacking
// || 10:30-11:00 | Everyone | Coffee break
// || 11:00-12:30 | Everyone | Hacking
// || *12:30-14:30* | *Everyone* | *Lunch*
// || 14:30-15:30 | Everyone | Hacking
// || 15:30-16:00 | Everyone | Coffee break
// || 16:00-17:00 | Everyone | Hacking
// |===


== *Registration and Practical Information*

*TBD*

////

*Conference pre-registration for speakers:* https://docs.google.com/forms/d/e/1FAIpQLScqCwRYuMFyZceYa7skzXrZ5feg9Y5nfURVs_d_Ug7K_BY3pQ/viewform?usp=header

*Registration for event attendees:* https://docs.google.com/forms/d/e/1FAIpQLScsS1eLIfaYFTmXpH8dUaGtWpud6HWEWKdStdahbMvuqiW9iQ/viewform?usp=header

////

Registration includes access to: all lectures and practical sessions, all course materials.

=== Cost

The cost includes all expenses associated with the event, with lunches, coffee, and snacks. In certain circumstances, we can waive the fees.

- Academic - $250, with financial support if needed
- Industry - $500

=== Selection criteria

This practical course / workshop is intended for biologists and bioinformaticians interested in studying organisms with high genetic diversity or without a reference genome, as well as those involved in comparative genomics and the assembly of pangenomes for any species.

Selection of participants will be based on:

- good knowledge of Linux operating system and basic shell commands. This will be a mandatory prerequisite.
- familiarity with genomics data formats (e.g., FASTA, VCF, BED, ...) is a plus.
- impact of the practical course / workshop for the participant and his/her research group.
- stage of the research project: priority will be given to participants with data already available and ready to be analyzed (participants data will not be analyzed during the practical course / workshop).

Fulfillment of these conditions by participants will be assessed through the registration form.

[cols=">a,<a,<a", frame=none, grid=none]
|===
| image::images/memphis.jpg[width=300,height=300]
|image::images/men.jpg[width=600,height=400]
| image::images/bridge.jpg[width=300,height=300]
|===

https://www.memphistravel.com/sports-outdoors?gclid=Cj0KCQiAgOefBhDgARIsAMhqXA4Gt_kloBAAqe5LDWNW3199TI8DzRrUo4fCqSQ_cKLkRPd4xv46TUgaAt4WEALw_wcB[memphistravel]

{empty} +
{empty} +

== *Contact*

- mailto:aguarra1@uthsc.edu[aguarra1@uthsc.edu]
